spisea/evolution.py

```diff
@@ -1615,6 +1615,91 @@ def get_orig_pisa_isochrones(metallicity=0.015):
 
     return data
 
+
+#==============================#
+# BPASS v2.2 models
+#==============================#
+#These models were published by Stanway and Elridge et. al in 2018
+#The code is based from the Merged Evolution model methods outlined in the . After all, I want some degree of consistency with regards to
+#Our original isochrone files are the stellar models (single, binary, and secondary) listed in the BPASS input files from one of the stellar models for ages 10**6.0, 10**6.1, ... , 10**11.0 years.
+
+class BPASS (StellarEvolution):
+    def __init__(self):
+        #Note: All of the possible mass list items include possible mass for single star, binary primary and secondary.
+        self.age_list=[round(6.0+x*0.1,1) for x in range(0, 51)]
+        #Possible metallicities for BPASS models
+        self.z_list=[10**-5, 10**-4,10**-3, 0.002, 0.003, 0.004, 0.006, 0.008, 0.010, 0.014, 0.020, 0.030, 0.040]
+        self.models_dir='/g/lu/models/evolution/BPASS/v2.2/'
```

Comment:
The directory shouldn't be hard-coded... use the environment variable as is done in the other StellarEvolution objects.